docs/quickstart.md

```diff
@@ -79,7 +79,7 @@ In this section, we'll use a sample python script to submit a federated learning
     python -m pip install -r ./examples/pipelines/fl_cross_silo_literal/requirements.txt
     ```
 
-2. To connect to your newly created Azure ML workspace, we'll need to create a `config.json` file at the root of this repo. Follow the instructions on how to get this from the [Azure ML documentation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-configure-environment#workspace).
+2. To connect to your newly created Azure ML workspace, you can provide the following values as CLI arguments or can create a `config.json` file at the root of this repo. Follow the instructions on how to get this from the [Azure ML documentation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-configure-environment#workspace).
```

Comment:
Let's make it easier by just removing the mention of `config.json` here, and move it as a "note" after we provide the CLI version

Reply:
so if someone just goes through, they'll use cli args, period. but if they're curious they can use config.json instead, but this is a side note, not the main quicktart.

Reply:
Gottu!